metadata-models/src/main/pegasus/com/linkedin/datajob/datahub/IngestionCheckpointState.pdl

```diff
@@ -0,0 +1,26 @@
+namespace com.linkedin.datajob.datahub
+
+import com.linkedin.timeseries.TimeseriesAspectBase
+import com.linkedin.timeseries.TimeWindowSize
+
+/**
+ * The checkpoint state object of a datahub ingestion run for a given job.
+ */
+record IngestionCheckpointState {
+
+  /**
+   * The version of the state format.
+   */
+  formatVersion: string
+
+  /**
+   * The serialization/deserialization protocol.
+   */
+  serde: string
+
+  /**
+   * Opaque blob of the state representation.
+   */
+  byte_payload: optional bytes
```

Comment:
convert to camel case ... or drop the `byte_` prefix.

Reply:
camel-case option would be `payloadInBytes`

Reply:
I would recommend just `payload`

Reply:
looked perfect on the python side, making the change ;-)